doc-content/enterprise-only/getting-started/data-model-proc.adoc

```diff
@@ -1,51 +1,72 @@
+. Click *Save*.
+
+== Creating the Property data object
+Follow these steps to create the *Property* data object, which contains information about the property details, such as the property age and price.
 
 .Procedure
-. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *Mortgage_Process*.
-. Enter `Application.java` in the project's asset search box, press Enter, and click the *Application* data object.
-. Review the *Application* data object values.
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *mortgage-process*.
+. Click *Add Asset* and select *Data Object*.
+. Enter `Property` in the *Data Object* field of the *Create new Data Object* window.
+. Select `com.myspace.mortgage_process` from the *Package* pull-down menu and click *Ok*.
+. Enter `Property` in the *Label* field of the *'Property'- general properties* section.
+. Click *+add field* and input the following *Property* data object values. Click *Create and continue* after each addition. For the last addition, click *Create*.
 +
-image::getting-started/field-data-application.png[]
+image::getting-started/field-data-property.png[]
+
+. Click *Save*.
 
-== Viewing the Property data object
-Follow these steps to familiarize yourself with the predefined *Property* data object, which contains information about the property details, such as the property age and price.
+== Creating the ValidationErrorDO data object
+Follow these steps to create the *ValidationErrorDO* data object, which specifies the cause of an application error.
 
 .Procedure
-. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *Mortgage_Process*.
-. Enter `Property.java` in the project's asset search box, press Enter, and click the *Property* data object.
-. Review the *Property* data object values.
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *mortgage-process*.
+. Click *Add Asset* and select *Data Object*.
+. Enter `ValidationErrorDO` in the *Data Object* field of the *Create new Data Object* window.
+. Select `com.myspace.mortgage_process` from the *Package* pull-down menu and click *Ok*.
+. Enter `ValidationErrorDO` in the *Label* field of the *'ValidationErrorDO'- general properties* section.
+. Click *+add field* and input the following *ValidationErrorDO* data object values. Click *Create and continue* after each addition. For the last addition, click *Create*.
 +
-image::getting-started/field-data-property.png[]
+image::getting-started/field-data-val-error.png[]
+
+. Click *Save*.
 
-== Viewing the ValidationErrorDO data object
-Follow these steps to familiarize yourself with the predefined *ValidationErrorDO* data object, which specifies the cause of an application error.
+== Creating the Application data object
+Follow these steps to create the *Application* data object, which contains information about the mortgage details, such as the down payment and the mortgage amount.
```

Comment:
mortage, such as 

or contains detailed information about the mortgage

Reply:
This section has been reworked to have them actually create instead instead of view the DOs